std/utf.d

```diff
@@ -4186,6 +4186,7 @@ private int impureVariable;
  *
  * Params:
  *      C = `char`, `wchar`, or `dchar`
+ *      useReplacementDchar = if invalid UTF, return replacementDchar rather than throwing
```

Comment:
Invert this (`If No.useReplacementDchar gives, throw rather than return replacementDchar on invalid UTF`), or update the documentation block above. Currently it's worded as an additional behavior to the behavior described in the second paragraph.